docs/Supported-Versions.md

```diff
@@ -14,9 +14,9 @@ We currently support the following versions of mobile OS:
 
 On iOS:
 
-- The used biometric type depends on the particular device capabilities.
+- The used biometric type depends on the particular device's capabilities.
 
 On Android:
 
-- Since Android 6.0, we offer the biometric authentication via the fingerprint authentication support.
-- Since Android 9.0, we offer the biometric authentication via the newly introduced unified biometric authentication dialog. However, we had to fallback to the old fingerprint authentication on several devices where the new biometric support is broken (as a well-known issue).
+- Since Android 6.0, we have offered biometric authentication via fingerprint authentication support.
+- Since Android 9.0, we offer biometric authentication via the newly introduced unified biometric authentication dialog. However, we had to fallback to the old fingerprint authentication on several devices where the new biometric support was broken (as a well-known issue).
```

Comment:
```suggestion
- Since Android 9.0, we have offered biometric authentication via the newly introduced unified biometric authentication dialog. However, we had to fallback to the old fingerprint authentication on several devices where the new biometric support was broken (as a well-known issue).
```